model/cat/pit_segments/structure.smithy

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+
+@input
+structure GetAllPitSegments_Input {
+}
+
+@output
+structure GetAllPitSegments_Output {
+    @documentation("Information about the active point-in-time segments.")
+    content: PitSegments
+}
+
+@input
+structure GetPitSegments_Input{
+    pit_id: PitIds
```

Comment:
`pit_id` is a required field according to the documentation. Lets annotate it with `@required`? Also, is this the request body, if so it would be marked as `@httpPayload`.